diff --git a/node_modules/web3-eth-iban/README.md b/node_modules/web3-eth-iban/README.md
index 0c3f27e..d540ab0 100644
--- a/node_modules/web3-eth-iban/README.md
+++ b/node_modules/web3-eth-iban/README.md
@@ -10,20 +10,12 @@ Please read the [documentation][docs] for more.
 
 ## Installation
 
-You can install the package either using [NPM](https://www.npmjs.com/package/web3-eth-iban) or using [Yarn](https://yarnpkg.com/package/web3-eth-iban)
-
-### Using NPM
+### Node.js
 
 ```bash
 npm install web3-eth-iban
 ```
 
-### Using Yarn
-
-```bash
-yarn add web3-eth-iban
-```
-
 ## Usage
 
 ```js
diff --git a/node_modules/web3-eth-iban/types/index.d.ts b/node_modules/web3-eth-iban/types/index.d.ts
index 75c8dae..6890787 100644
--- a/node_modules/web3-eth-iban/types/index.d.ts
+++ b/node_modules/web3-eth-iban/types/index.d.ts
@@ -58,5 +58,3 @@ export interface IndirectOptions {
     institution: string;
     identifier: string;
 }
-
-export default Iban
diff --git a/node_modules/web3-eth-iban/types/tests/iban-tests.ts b/node_modules/web3-eth-iban/types/tests/iban-tests.ts
index 71e5080..41c921d 100644
--- a/node_modules/web3-eth-iban/types/tests/iban-tests.ts
+++ b/node_modules/web3-eth-iban/types/tests/iban-tests.ts
@@ -20,14 +20,11 @@
  * @date 2018
  */
 
-import IbanDefault, { Iban, IndirectOptions } from 'web3-eth-iban';
+import { Iban, IndirectOptions } from 'web3-eth-iban';
 
 const iban = 'XE7338O073KYGTWWZN0F2WZ0R8PX5ZPPZS';
 const address = '0x45cd08334aeedd8a06265b2ae302e3597d8faa28';
 
-// $ExpectType Iban
-const iban_default = new IbanDefault(iban);
-
 // $ExpectType Iban
 const ibanClass = new Iban(iban);
 
